package.json

```diff
@@ -152,11 +161,16 @@
       "extraResources": [
         "src/practices",
         "mac-screenlock-check/screenlock-check"
-      ]
+      ],
+      "notarize": false
```

Comment:
why not notarize? 🤔

Reply:
By default, the upgraded version of electron-builder does the notarize. we just disabled that.